components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/OAuthUtil.java

```diff
@@ -297,6 +297,22 @@ public static void clearOAuthCache(String consumerKey, User authorizedUser, Stri
     public static void clearOAuthCache(String consumerKey, AuthenticatedUser authorizedUser, String scope,
                                        String tokenBindingReference) {
 
+        clearOAuthCache(consumerKey, authorizedUser, scope, tokenBindingReference,
+                OAuthConstants.AuthorizedOrganization.NONE);
+    }
+
+
+    /**
+     * Clear OAuth cache based on the application, authorized user, scope list and token binding reference.
+     *
+     * @param consumerKey           Client id of the application the token issued to.
+     * @param authorizedUser        Authorized user.
+     * @param scope                 Scope list.
+     * @param tokenBindingReference Token binding reference.
```

Comment:
authorizedOrganization param in not described in the comment

Reply:
Fixed by - https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2187/commits/377c45e4e0656a621e117163f96d44f2c2432f91